src/community/__stories__/blocks-story.tsx

```diff
@@ -170,8 +176,10 @@ BlockHighlightedValue.args = {
     text: 'text',
     value: '20',
     secondaryValue: '20',
+    Pretitle: 'Pretitle',
```

Comment:
```suggestion
    pretitle: 'Pretitle',
```

uppercase inconsistency

fix in the rest of the uses